mcs/packages/Makefile

```diff
@@ -3,7 +3,7 @@ include ../build/rules.make
 
 ROSLYN_CSC_DIR = $(dir $(CSC_LOCATION))
 
-ROSLYN_FILES = \
+ROSLYN_FILES_FOR_MONO = \
```

Comment:
I think we should copy only real msbuild file (e.g. *.targets, VBCSCompiler.*, Microsoft.Build.Tasks.CodeAnalysis.dll)

The rest is either already in lib/mono/4.5/ or we should install it (e.g csi.exe + csi)

Thoughts?